events/lastAvatarStandingTrivia/entityServerScripts/triviaMasterServer.js

```diff
@@ -39,9 +39,9 @@
         "rezValidator",
         "deleteValidator",
         "checkAnswer",
-        "loseCoins",
+        // "loseCoins",
         "stopSound",
-        "winCoins",
+        // "winCoins",
```

Comment:
remove comment

Reply:
This is still needed for trivia in the trivia domain, it was only removed for multi-con

Reply:
Remove and see comment